packages/theming/atlas/src/themesource/atlas_core/web/main.scss

```diff
@@ -21,113 +23,138 @@
 );
-@include include-widget-styles("core/widgets/modal", $exclude-modal);
+@include include-widget-styles("core/widgets/modal", map.get(exclusions.$exclusions, "exclude-modal"));
 @include include-widget-styles(
     "core/widgets/navigation-bar",
-    $exclude-navigation-bar,
+    map.get(exclusions.$exclusions, "exclude-navigation-bar"),
     "core/helpers/navigation-bar",
-    $exclude-navigation-bar-helpers
+    map.get(exclusions.$exclusions, "exclude-navigation-bar-helpers")
+);
+@include include-widget-styles(
+    "core/widgets/navigation-list",
+    map.get(exclusions.$exclusions, "exclude-navigation-list")
 );
-@include include-widget-styles("core/widgets/navigation-list", $exclude-navigation-list);
 @include include-widget-styles(
     "core/widgets/navigation-tree",
-    $exclude-navigation-tree,
+    map.get(exclusions.$exclusions, "exclude-navigation-tree"),
     "core/helpers/navigation-tree",
-    $exclude-navigation-tree-helpers
+    map.get(exclusions.$exclusions, "exclude-navigation-tree-helpers")
 );
-@include include-widget-styles("core/widgets/pop-up-menu", $exclude-pop-up-menu);
+@include include-widget-styles("core/widgets/pop-up-menu", map.get(exclusions.$exclusions, "exclude-pop-up-menu"));
 @include include-widget-styles(
     "core/widgets/simple-menu-bar",
-    $exclude-simple-menu-bar,
+    map.get(exclusions.$exclusions, "exclude-simple-menu-bar"),
     "core/helpers/simple-menu-bar",
-    $exclude-simple-menu-bar-helpers
+    map.get(exclusions.$exclusions, "exclude-simple-menu-bar-helpers")
 );
-@include include-widget-styles("core/widgets/radio-button", $exclude-radio-button);
+@include include-widget-styles("core/widgets/radio-button", map.get(exclusions.$exclusions, "exclude-radio-button"));
 @include include-widget-styles(
     "core/widgets/tab-container",
-    $exclude-tab-container,
+    map.get(exclusions.$exclusions, "exclude-tab-container"),
     "core/helpers/tab-container",
-    $exclude-tab-container-helpers
+    map.get(exclusions.$exclusions, "exclude-tab-container-helpers")
+);
+@include include-widget-styles(
+    "core/widgets/table",
+    map.get(exclusions.$exclusions, "exclude-table") "core/helpers/table",
+    map.get(exclusions.$exclusions, "exclude-table-helpers")
 );
-@include include-widget-styles("core/widgets/table", $exclude-table, "core/helpers/table", $exclude-table-helpers);
 @include include-widget-styles(
     "core/widgets/template-grid",
-    $exclude-template-grid,
+    map.get(exclusions.$exclusions, "exclude-template-grid"),
     "core/helpers/template-grid",
-    $exclude-template-grid-helpers
+    map.get(exclusions.$exclusions, "exclude-template-grid-helpers")
 );
 @include include-widget-styles(
     "core/widgets/typography",
-    $exclude-typography,
+    map.get(exclusions.$exclusions, "exclude-typography"),
     "core/helpers/typography",
-    $exclude-typography-helpers
+    map.get(exclusions.$exclusions, "exclude-typography-helpers")
```

Comment:
We still need to do the check in this file. Also, there were some more `@extend` uses in the styles that we need to check.